.github/workflows/release-script-tests.yml

```diff
@@ -23,4 +24,4 @@ jobs:
       - name: Install dependencies
         run: pip3 install -r requirements.txt
       - name: Run tests
-        run: pytest -v
+        run: pytest -vv
```

Comment:
What is the difference between `-v` and `-vv`?

Reply:
This is for the level of verbosity https://docs.pytest.org/en/7.1.x/how-to/output.html?highlight=verbose#verbosity
There are higher levels btw -vvv, -vvvv.
But yeah, this does not need to be -vv, I'll switch to -v again.